db/migrations/postgres/000095_upgrade_outgoing_webhooks.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE outgoingwebhooks ADD COLUMN IF NOT EXISTS enabled boolean;
```

Comment:
Can we add constraints for `NOT NULL` and `DEFAULT TRUE`?

Reply:
Also, the logic for outgoing webhooks isn't implemented here yet, just the sql scripts. This is probably already known but just mentioning it